R/aaa.R

```diff
@@ -69,6 +69,10 @@ the <- new.env(parent = emptyenv())
 #' `on_load()` is implemented via `.onLoad()` and requires
 #' `run_on_load()` to be called from that hook.
 #'
+#' The expressions inside `on_load()` do not undergo static analysis
+#' by `R CMD check`. Therefore, it is advisable to only use
+#' simple function calls inside `on_load()`.
```

Comment:
I think we could add support for on_load to other tools so I'd remove the latter part:

```suggestion
#' Note that the expressions inside `on_load()` do not undergo static analysis
#' by `R CMD check`.
```

Reply:
I don't understand "support ... to other tools", I'm fine with the change if it makes sense to you.

Reply:
On the other hand extracting to a function also makes sense in terms of style. Let's just merge as is.